lib/tournament.json

```diff
@@ -625,6 +625,44 @@
       }
     ]
   ],
+  "rankings": [
```

Comment:
The reason that I'm doing it this way is that the "teams" field always remains unchanged but "round-robin" and "rankings" get constantly updated. Ultimately the latter two will end up in a database whereas the former will be encoded in a file.